go/hack/runtime.go

```diff
@@ -0,0 +1,45 @@
+/*
+Copyright 2019 The Vitess Authors.
```

Comment:
nit: 2021! Don't make me think 2020 is still to come 😭

Reply:
Fixed!